e2e/install_test.go

```diff
@@ -1973,3 +1980,130 @@ func cleanupCluster(t *testing.T, tc *cluster.Output) {
 	generateAndCopySupportBundle(t, tc)
 	copyPlaywrightReport(t, tc)
 }
+
+func TestMultiNodeAirgapUpgradeLarge(t *testing.T) {
```

Comment:
Any chance this could be named something like "TestFiveNodeAirgapUpgrade" instead?

Reply:
Done.